features/atcryptoauth/MbedPlatform.h

```diff
@@ -0,0 +1,103 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2018 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef MBEDPLATFORM_H
+#define MBEDPLATFORM_H
+
+#include "mbed.h"
+#include "ATCAPlatformInterface.h"
+
+#if !defined(DEVICE_I2C)
+typedef void I2C;
+#endif /* DEVICE_I2C */
+
+/** Mbed implementation of abstract Platform interface.
+ */
+class MbedPlatform : public ATCAPlatformInterface {
```

Comment:
why this class has generic name `MbedPlatform` ? This does not look a generic platform class that can be used outside of this feature?